File: codeforces/low_difficulty/difficulty_900/4.even_odds_[318A]/EvenOdds.java
/* Author: José Rodolfo (jric2002) */
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
public class EvenOdds {
  public static void main(String[] args) throws IOException {
    Double n, k, temp, number;
    String[] values;
    BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
    values = (input.readLine().split(" "));
    n = Double.valueOf(values[0]);
    k = Double.valueOf(values[1]);
    temp = ((n % 2) == 0) ? (n / 2) : ((n + 1) / 2);
    number = (k <= temp) ? ((2 * k) - 1) : (2 * (k - temp));
    System.out.print(String.format("%.0f", number));
  }
}

File: codeforces/low_difficulty/difficulty_900/4.even_odds_[318A]/even_odds.c
#include <stdio.h>
/* Author: José Rodolfo (jric2002) */
int main() {
  long long int n, k, temp, number;
  scanf("%lli %lli", &n, &k);
  temp = ((n % 2) == 0) ? (n / 2) : ((n + 1) / 2);
  number = (k <= temp) ? ((2 * k) - 1) : (2 * (k - temp));
  printf("%lld\n", number);
  return 0;
}